index.html

```diff
@@ -1,85 +1,53 @@
-                <h5>vabuyia@gmail.com</h5>
-            </div>
-
-            <div class="box box2 ">
-
-                <div class="about">
-                    
-                    <h4>ABOUT ME</h4>
-                    <p>I was born in 1983. I live in Basra and work as a Software Developer/Architect. <br> 
-                        I’ve been a professional developer for more than 2 years. <br> 
-                        I’ve mostly worked professionally with HTML, CSS, JavaScript, Ruby, Perl and PHP,<br> 
-                        but also little bit with Java and Python. C, C++. 
-                    </p>                         
-                </div>
-
-                <div class="social-icon">
-
-                    <h4> CONTACT </h4>
-
-                    <a href="https://twitter.com/abuyiag" target="_blank"> 
-                        <ion-icon name="logo-twitter"></ion-icon>  
-                    </a>
-
-                    <a href="linkedin.com/in/vincent-abuya-a1940555" target="_blank"> 
-                        <ion-icon name="logo-linkedin"></ion-icon> 
-                    </a>   
-
-                    <a href="https://github.com/vabuyia" target="_blank"> 
-                        <ion-icon name="logo-github"></ion-icon>
-                    </a> 
-                    <a href="https://github.com/vabuyia" target="_blank"> 
-                        <ion-icon name="logo-github"></ion-icon>
-                    </a>
-                                        
-                </div> 
-
-            </div>
-
-        </div>
-
-                          
-    </div>
-
-        
-    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
-    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+<header>
+  <img src="wccl.png" alt=" " class="logo ">
+
+  <nav> 
+    <ul class= "nav-area">
+      <li> <a href="#" >Portfolio</a></li>
+      <li> <a href="#" >About</a></li>
+      <li> <a href="#" >Contact</a></li>
+    </ul>
+  </nav>
+
+</header>
+
+<div class= wrapper> 
+
+  <div class ="upper"> 
+    <h1>I'm Vincent </h1>  
+    <br>
+    <h1>Glad to see you! </h1>
+  </div>
+
+  <div class="lower">   
+    <p>I'm a software developer! i can help you build a product. feature or website Look through <br>
+      some of my work and experience! if you like what you see and have a project you needed <br>
+       coded. don't hesistate to contact me. </p>
+  </div>
+
+  <div class="connect">
+    <br>
+    <h3>LET CONNECT</h3>
+
+    <a href="https://twitter.com/abuyiag" target="_blank" rel="noopener noreferrer"> <ion-icon name="logo-twitter"></ion-icon> </a>;
+    <a href=" linkedin.com/in/vincent-abuya-a1940555" target="_blank" rel="noopener noreferrer"> <ion-icon name="logo-linkedin"> </ion-icon> </a>;
+    <a href="https://github.com/vabuyia" target="_blank" rel="noopener noreferrer"> <ion-icon name="logo-github"></ion-icon> </a>;
+
```

Comment:
- [ ] For the social media icon. Because it's a list it's better to wrap them in a `ul` and put each link in a `li`.